docs/Spark/extensibility/dependencies.md

```diff
@@ -11,162 +11,82 @@ tags:
 
 Dependencies allow you to make use of third-party or custom code in your data Pipelines and Jobs. Dependencies can be
 written in Java, Scala, or PySpark, and can be connected to your data Pipelines by pointing
-to [Maven](https://mvnrepository.com/) or [PyPi](https://pypi.org/) coordinates.
+to [Maven](https://mvnrepository.com/) or [PyPI](https://pypi.org/) coordinates.
 
 ## Storage
 
-![See Dependencies](img/dependencies-storage.png)
+Dependencies can be stored at the Project or Pipeline level. Project-level dependencies are available to all Pipelines in the Project.
 
-All of your dependencies are stored at the Project-level. When adding a dependency to a single
-Pipeline, by default it becomes available to all the other Pipelines within the same Project. However, within each
-Pipeline you can control if a given dependency should be enabled or disabled.
+Dependencies are natively saved within your build system files. For example, if you're using Scala, dependencies are saved in the
+**pom.xml** file. If you're using Python, dependencies are saved in the **setup.py** file.
```

Comment:
we should put a tooltip to recommend that users store dependencies at a Project level whenever possible to improve the experience of interactive development. This will minimize the time spent connecting to clusters when switching between pipelines in the IDE.